<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User_dashboardWidget</class>
 <widget class="QWidget" name="User_dashboardWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>850</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1350</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#User_dashboardWidget{
	background-color: #f2efff;
}</string>
  </property>
  <widget class="QFrame" name="border_blueLines1">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-1</y>
     <width>1371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#border_blueLines1{
	background-color: #141c52;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="title_bodyDashboardSystemTitle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>1301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#title_bodyDashboardSystemTitle {
	background-color: transparent;
	color: #white;
}
</string>
    </property>
    <property name="text">
     <string>MaPro: Marigondon Records and Statistics System</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="border_blueLines2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>820</y>
     <width>1381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#border_blueLines2{
	background-color: #141c52;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="baseBodyFrame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>1311</width>
     <height>741</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#baseBodyFrame {
	background-color: #e7e2ff;
	border: none;
	border-radius: 10px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="logout_buttonLogout_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>740</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>8</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#logout_buttonLogout {
    background-color: #bc1c1c;
    color: white;
    border-radius: 5px;
    padding: 10px; /* Optional: Adjusts spacing inside the button */
}
#logout_buttonLogout:hover {
    background-color: #d82020; /* Slightly lighter blue when hovered */
}
#logout_buttonLogout:pressed {
    background-color: #8c1515; /* Darker blue when clicked */
}</string>
    </property>
    <property name="text">
     <string>LOGOUT</string>
    </property>
   </widget>
   <widget class="QLabel" name="title_bodyStatistics">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#title_bodyStatistics {
	background-color: transparent;
	color: #141c52;
}
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;STATISTICS: &lt;span style=&quot; color:#3600a1;&quot;&gt;INFRASTRUCTURES&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QFrame" name="statistics_frameBody">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>1271</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#statistics_frameBody{
	background-color: #f2efff;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QTableWidget" name="infra_table">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>551</width>
       <height>421</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#infra_table{
	background-color: #fcfcfc;
}</string>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>137</number>
     </attribute>
     <column>
      <property name="text">
       <string>Sitio Name</string>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="background">
       <color>
        <red>59</red>
        <green>57</green>
        <blue>100</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Public</string>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="background">
       <color>
        <red>59</red>
        <green>57</green>
        <blue>100</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Private</string>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="background">
       <color>
        <red>59</red>
        <green>57</green>
        <blue>100</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total Infrastructures</string>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="background">
       <color>
        <red>59</red>
        <green>57</green>
        <blue>100</blue>
       </color>
      </property>
     </column>
    </widget>
    <widget class="QFrame" name="demo_frameOthers">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>540</y>
       <width>371</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#demo_frameOthers{
	background-color: #fcfcfc;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="total_Partnership">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#total_Partnership {
	background-color: transparent;
	color: #4f4f4f;
}
</string>
      </property>
      <property name="text">
       <string>###,###</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="total_Sole">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#total_Sole {
	background-color: transparent;
	color: #4f4f4f;
}
</string>
      </property>
      <property name="text">
       <string>###,###</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_Sole">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#label_Sole {
	background-color: transparent;
	color: #4f4f4f;
}
</string>
      </property>
      <property name="text">
       <string>Public Infrastructures:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_Partnership">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#label_Partnership {
	background-color: transparent;
	color: #4f4f4f;
}
</string>
      </property>
      <property name="text">
       <string>Private  Infrastructures:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_desc_edlev_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>371</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#label_desc_edlev_2{
	color: #969696;
}</string>
      </property>
      <property name="text">
       <string>TOTAL NUMBER OF THE FOLLOWING:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="table_titleFrameStatistics">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>551</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#table_titleFrameStatistics{
	background-color: rgba(20, 28, 82, 128)
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="statistics_labelNote">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>541</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#statistics_labelNote {
	background-color: transparent;
	color: #FFFFFF;
}
</string>
      </property>
      <property name="text">
       <string>INFRASTRUCTURE PER SITIO</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="table_titleFrameStatistics3_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>371</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#table_titleFrameStatistics3_2{
	background-color: rgba(20, 28, 82, 128)
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="statistics_labelNote_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#statistics_labelNote_5 {
	background-color: transparent;
	color: #FFFFFF;
}
</string>
      </property>
      <property name="text">
       <string>PUBLIC &amp; PRIVATE DISTRIBUTION</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QTableWidget" name="infra_table_2">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>60</y>
       <width>551</width>
       <height>421</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#infra_table_2{
	background-color: #fcfcfc;
}</string>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>137</number>
     </attribute>
     <column>
      <property name="text">
       <string>Infrastructure Type</string>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="background">
       <color>
        <red>59</red>
        <green>57</green>
        <blue>100</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Public</string>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="background">
       <color>
        <red>59</red>
        <green>57</green>
        <blue>100</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Private</string>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="background">
       <color>
        <red>59</red>
        <green>57</green>
        <blue>100</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total Number</string>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="background">
       <color>
        <red>59</red>
        <green>57</green>
        <blue>100</blue>
       </color>
      </property>
     </column>
    </widget>
    <widget class="QFrame" name="table_titleFrameStatistics_2">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>551</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#table_titleFrameStatistics_2{
	background-color: rgba(20, 28, 82, 128)
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="statistics_labelNote_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>551</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#statistics_labelNote_2 {
	background-color: transparent;
	color: #FFFFFF;
}
</string>
      </property>
      <property name="text">
       <string>INFRASTRUCTURE TYPE COUNT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QToolButton" name="btn_returnToStatisticsPage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_returnToStatisticsPage {
    background-color: #f2efff;
    color: #141c52;
	border: 2px solid #141c52; 
	border-radius: 18px;
	text-align: left;
    padding: 10px; /* Optional: Adjusts spacing inside the button */
}
#btn_returnToStatisticsPage:hover {
    background-color: #fdfdff; /* Slightly lighter blue when hovered */
}
#btn_returnToStatisticsPage:pressed {
    background-color: #e7e5f4; /* Darker blue when clicked */
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Assets/FuncIcons/img_return.png</normaloff>../../../Assets/FuncIcons/img_return.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="filter_by_year">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#filter_by_year {
  background-color: #f8f9fa; /* Very light grey background */
  color: #495057; /* Medium grey text for readability */
  border: 1px solid #ced4da; /* Light grey border */
  padding: 10px 15px; /* Comfortable padding */
  border-radius: 6px; /* Slightly rounded corners - a modern touch */
  font-size: 0.9rem; /* Slightly smaller, modern font size */
  line-height: 1.5; /* Improved line height for better text spacing */
  box-shadow: 0 1px 3px rgba(0, 0, 0, 0.05); /* Subtle shadow for depth */
}</string>
    </property>
    <property name="placeholderText">
     <string>PROFILING YEAR</string>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>2025</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2024</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>All Year</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
